app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/DataOverViewActivityTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.github.HumanLearning2021.HumanLearningApp
+
+import android.content.Intent
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onData
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.intent.rule.IntentsTestRule
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.github.HumanLearning2021.HumanLearningApp.view.DisplayDatasetActivity
+import com.github.HumanLearning2021.HumanLearningApp.view.DisplayImageActivity
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.After
+import org.junit.Before
+
+
+@RunWith(AndroidJUnit4::class)
+
+class DataOverViewActivityTest {
```

Comment:
```suggestion
class DatasetOverviewActivityTest {
```
Would be better to have a more precise name. (also overview is one word https://www.wordreference.com/enfr/overview)